spec/helpers/sign_download_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe SignsHelper, type: :helper do
+  drawing_1 = "1234/a_test-default"
+
+  describe '#convert_to_high_resolution' do
+    context 'when provided with a "default" sign drawing' do
+      it 'returns the sign drawing with the text "high_resolution" as part of the file name' do
+        expect(helper.convert_to_high_resolution(drawing_1)).to include("high_resolution")
+      end
+    end
+  end
```

Comment:
maybe an additional test for this helper to assert that if "default" isn't present in the sign name then it nothing is modified and the returned value is the same is what gets passed in?

Reply:
@danieljauch @Rabid-Dan yes I agree with you both. I won't be making changes today but it will get done. Cheers

Reply:
@danieljauch @Rabid-Dan currently I have a bit of spare time so thought I could write up some thoughts re swapping the 'default' for 'high_resolution'. Thoughts as follows.
- currently all drawing images have the 'default' text appended.
- does it matter if there are 1 2 or more instances of `default` in the drawing sign string?
- if there are drawing signs like `1234/watermelon-default`  or `1235/default-watermelon-default` or `1236/watermelon-default-default` and they are split into an array, then I can use a map method to change the last element to '-high_resolution`.
- when it comes to the time that users may be able to upload their own drawings, I think this will come with a different set of logic - like validating file path, checking the file extension, adding 'default' etc: